packages/@okta/i18n/src/properties/login.properties

```diff
@@ -509,30 +511,38 @@ enroll.webauthn.instructions.noSupportForBiometric = Note: Some browsers may not
 authfactor.webauthn.error = You are currently unable to use a Security key or biometric authenticator. Try again.
 
 ## Enter Passcode Form
-enroll.totp.enterCode = Enter code displayed from the application
+enroll.totp.enterCode = Enter temporary code generated from the application
 
 ## Barcode View
 # {0} is the name of a MFA factor
 enroll.totp.setupApp = Launch {0} on your mobile device and select “Add an account”. Scan the QR code to continue.
 # {0} is the name of a MFA factor
 enroll.totp.setupGoogleAuthApp = Launch {0}, tap the "+" icon, then select "Scan barcode".
 enroll.totp.cannotScan = Can't scan?
-enroll.totp.aria.cannotScan = Setup without scanning a QR code.
+enroll.totp.setupWithoutQRCode = Setup without scanning a QR code.
 enroll.totp.refreshBarcode = Refresh code
 
 ## Manually Setup Push
-enroll.totp.cannotScanBarcode = Can't scan barcode?
+enroll.totp.moreOptions = More options
 enroll.totp.manualSetupInstructions = To set up manually enter your Okta Account username and then input the following in the Secret Key Field
 enroll.totp.manualSetupInstructions.generic = To set up manually enter your Account username and then input the following in the Secret Key Field
 # {0} maps the the configured 'brandName', like "Okta".
 enroll.totp.manualSetupInstructions.specific = To set up manually enter your {0} Account username and then input the following in the Secret Key Field
 enroll.totp.sharedSecretInstructions = Enter your Okta Account username and enter the following in the Secret Key Field
-enroll.totp.sharedSecretInstructions.generic = Enter your Account username and enter the following in the Secret Key Field
-# {0} maps the the configured 'brandName', like "Okta".
-enroll.totp.sharedSecretInstructions.specific = Enter your {0} Account username and enter the following in the Secret Key Field
+enroll.totp.sharedSecretInstructions.aria.intro= Please follow below instructions to setup Key.
+# {0} maps the the configured Factor name, like "Okta Verify" , "Google Authenticator".
+enroll.totp.sharedSecretInstructions.step1 = On your mobile device, download the {0} app from the App Store® (iPhone® and iPad®) or on Google Play (Android™ devices).
```

Comment:
Curious why the registered trademark indicator is used for only this message. If we're going to use this here, shouldn't it be everywhere?